### PR TITLE
109 adapt all examples and tests to datasets

### PR DESCRIPTION
**Pull request by MikeJSeo**
also fixed test examples!

### CODE REVIEW

File: inst/examples/basic_kmplot2_ex.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+data(adtte_sat)
+data(pseudo_ipd_sat)
+
```
Comment (gravesti) [inline inst/examples/basic_kmplot2_ex.R:3]: I think you need `library(survival)` here

File: inst/examples/basic_kmplot_ex.R [hunk 2]
```diff
@@ -1,6 +1,8 @@
-library(survival)
-load(system.file("extdata", "combined_data_tte.rda", package = "maicplus", mustWork = TRUE))
-kmobj <- survfit(Surv(TIME, EVENT) ~ ARM, combined_data_tte, conf.type = "log-log")
+data(adtte_sat)
```
Comment (gravesti) [inline inst/examples/basic_kmplot_ex.R:1]: I think you need `library(survival)` here

File: tests/testthat/test-maic_anchored.R [hunk 3]
```diff
@@ -89,184 +34,23 @@ test_that("maic_anchored works for TTE using robust SE", {
...
+    c("7.6[6.3;10.3]", "1.8[1.6; 2.0]", "12.2[10.2; NA]", " 1.8[ 1.5;2.3]", "2.7[2.3;3.3]", "1.9[1.7;2.0]", "--")
   )
   expect_equal(
     result$inferential$report_overall_robustCI$`HR[95% CI]`,
-    c("0.22[0.19;0.26]", "", "0.16[0.11;0.24]", "", "0.57[0.48;0.68]", "", "0.29 [0.19; 0.44]")
-  )
-})
-
-
-test_that("maic_anchored works for TTE using bootstrap SE", {
```
Comment (gravesti) [inline tests/testthat/test-maic_anchored.R:114]: No test on the bootstrap results anymore?

File: inst/examples/basic_kmplot2_ex.R (hunk 1 above)
Comment (MikeJSeo) [reply inst/examples/basic_kmplot2_ex.R:3]: isn't survival already imported when importing maicplus?

File: tests/testthat/test-maic_anchored.R (hunk 3 above)
Comment (MikeJSeo) [reply tests/testthat/test-maic_anchored.R:114]: oh I deleted some parts.. I'll put them back in

File: inst/examples/basic_kmplot2_ex.R (hunk 1 above)
Comment (gravesti) [reply inst/examples/basic_kmplot2_ex.R:3]: The package imports `survival` meaning that we can use `survival` functions in our package functions. It does not make make those functions available to the user the same as `library(survival)` does. (ie load and attaching the namespace).

To do that we would need to put `survival` in `depends:`, but this is not the modern way.

File: R/time-helper.R [hunk 4]
```diff
@@ -68,7 +67,8 @@ get_time_conversion <- function(factor = c("days", "weeks", "months", "years"))
 #' @param as A time scale to convert to. One of "days", "weeks", "months", "years"
 #'
 #' @return Returns a numeric vector calculated from `times / get_time_conversion(factor = as)`
-
+#' @examples
```
Comment (gravesti) [inline R/time-helper.R:70]:
```suggestion
#' @export
#' @examples
```

File: tests/testthat/test-maic_anchored.R [hunk 5]
```diff
@@ -255,18 +145,18 @@ test_that("maic_anchored for binary case gives the expected result", {
...
+    c("390(78.0)", "338(67.6)", "128.8(25.8)", "124.2(24.8)", "280(58.3)", "120(37.5)", "--")
   )
   expect_equal(
     result$inferential$report_overall_robustCI$N,
     c("500", "500", "500", "500", "480", "320", "--")
   )
   expect_equal(
     result$inferential$report_overall_bootCI$`OR[95% CI]`,
-    c("1.70[1.28;2.26]", "", "1.46[0.63;1.58]", "", "2.33[1.75;3.12]", "", "0.63 [0.27; 0.68]")
+    c("1.70[1.28;2.26]", "", "1.14[0.33;0.98]", "", "2.33[1.75;3.12]", "", "0.49 [0.14; 0.42]")
```
Comment (gravesti) [inline tests/testthat/test-maic_anchored.R:160]: @MikeJSeo with this new data, the bootstrap has worse bias than the previous!
` "0.63 [0.27; 0.68]"` vs `"0.49 [0.14; 0.42]"`

**Review by gravesti — APPROVED** [review]

**Pull request merged**